Lukas' changes

skogsbaer opened the pull request.

lukasepple commented:
I fixed the issues I had. The Generics-derived instances seem to be unique (see test properties). Hashing Integers now completely works and split uses `Word64`s.

skogsbaer commented:
great!

sternenseemann <notifications@github.com> schrieb am Di., 5. Jan. 2016 um
11:50 Uhr:

> I fixed the issues I had. The Generics-derived instances seem to be unique
> (see test properties). Hashing Integers now completely works and split uses
> Word64s.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/factisresearch/large-hashable/pull/1#issuecomment-168970354>
> .
>
